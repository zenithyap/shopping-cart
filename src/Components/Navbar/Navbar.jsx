import { Link } from 'react-router-dom';
import styles from './navbar.module.css';

export default function Navbar() {
  return (
    <div className={styles.navbar}>
      <Link to="/">Home</Link>
      <Link to="shop">Shop</Link>
      <Link to="cart">Cart</Link>
    </div>
  );
}
