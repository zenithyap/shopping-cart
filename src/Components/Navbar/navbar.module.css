.navbar {
  display: flex;
  height: 70px;
  align-items: center;
  justify-content: right;
  gap: 2rem;
  padding-inline: 3rem;
}
