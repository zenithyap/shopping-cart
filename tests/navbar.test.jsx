import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/react'
import { createBrowserRouter, RouterProvider } from 'react-router-dom'
import Navbar from '../src/Components/NavBar'

describe('Navbar component', () => {
    it('Navbar should render correctly', () => {
        const router = createBrowserRouter([
            {
                path: '/',
                element: <Navbar />,
            },
        ])
        const { container } = render(<RouterProvider router={router} />);
        expect(container).toMatchSnapshot();
    });
});