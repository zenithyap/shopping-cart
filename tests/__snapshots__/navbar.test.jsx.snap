// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar component > Navbar should render correctly 1`] = `
<div>
  <div
    class="_navbar_d78bf3"
  >
    <a
      data-discover="true"
      href="/"
    >
      Home
    </a>
    <a
      data-discover="true"
      href="/shop"
    >
      Shop
    </a>
    <a
      data-discover="true"
      href="/cart"
    >
      Cart
    </a>
  </div>
</div>
`;
